app/dashboard/tip_views.py

```diff
@@ -135,12 +147,13 @@ def receive_tip_legacy(request):
 @csrf_exempt
 @ratelimit(key='ip', rate='2/m', method=ratelimit.UNSAFE, block=True)
 def receive_tip_v2(request, pk, txid, network):
-    """Handle the receiving of a tip (the POST).
-
+    """Handle the receiving of a tip (the POST)
+    TODO: Deprecate after v3 has been live for a month
     Returns:
-        TemplateResponse: The UI with the tip confirmed.
+        TemplateResponse: the UI with the tip confirmed
```

Comment:
`257` - docstrings should end in punctuation.  This will throw off any docstring linter... not sure why we removed punctuation?

Reply:
updated